s3tests/functional/test_headers.py

```diff
@@ -199,6 +200,7 @@ def test_object_create_bad_contentlength_none():
 @attr(assertion='fails 400')
 @nose.with_setup(teardown=_clear_custom_headers)
 @attr('fails_on_rgw')
+@attr('fails_on_dbstore')
```

Comment:
since 'dbstore' implies 'rgw', can we avoid tagging stuff that already has 'fails_on_rgw'?

this way, the 'fails_on_dbstore' tag means that something needs fixing in dbstore itself, and we can hunt those down systematically until nothing is left

to run against dbstore, you'd just use both '!fails_on_rgw,!fails_on_dbstore'

Reply:
Agreed, I tried to tag all of the tests on here that fall into this category but quickly realized there are dozens of tests that `fail_on_rgw`.

Reply:
Done.